class ValidationError(Exception):
    pass

class ContainmentError(Exception):
    pass
